configs/common/Options.py

```
@@ -672,3 +672,38 @@ def addXiangshanFSOptions(parser):
+                       help='Checkpoint interval for trace rollback (default: 64)')
+    parser.add_argument('--trace-disable-bp-validation', action='store_true',
+                       help='Disable branch predictor validation against trace')
+    parser.add_argument('--trace-mispredict-penalty', type=int, default=8,
+                       help='Cycles to penalize on mispredict (default: 8)')
+    parser.add_argument('--trace-disable-wrongpath', action='store_true',
+                       help='Disable explicit wrong-path injection (use stall model)')
+    parser.add_argument('--trace-wrongpath-use-traceinst', action='store_true',
+                       help='Wrong-path injection uses trace instructions with checkpoint/restore (default: NOPs)')
+
+    # Check for trace mode before parsing to make generic-rv-cpt conditional
+    if '--enable-trace-mode' in sys.argv:
```

Ruff (0.14.8)

`sys` may be undefined, or defined from star imports

(F405)